src/js/extension.js

```diff
@@ -2,38 +2,35 @@
  * Not strictly necessary but keeps everything contained in one file.
  */
 
-export default {
-  storage: {
-    getLocal: (key, _default=null) => {
-      return browser.storage.local
-        .get({[key]: _default})
-        .then(obj => obj[key]);
-    },
-
-    setLocal: (key, val) => browser.storage.local.set({[key]: val}),
+export const storage = {
+  getLocal: (key, _default=null) => {
+    return browser.storage.local
+      .get({[key]: _default})
+      .then(obj => obj[key]);
   },
+  setLocal: (key, val) => browser.storage.local.set({[key]: val}),
+};
 
-  runtime: {
-    sendMessage: (msg) => browser.runtime.sendMessage(msg),
-
-    onInstalled: (cb) => browser.runtime.onInstalled.addListener(cb),
-    onMessage: (cb) => browser.runtime.onMessage.addListener(cb),
-  },
+export const runtime = {
+  sendMessage: (msg) => browser.runtime.sendMessage(msg),
+  onInstalled: (cb)  => browser.runtime.onInstalled.addListener(cb),
+  onMessage:   (cb)  => browser.runtime.onMessage.addListener(cb),
+};
 
-  tabs: {
-    create: (url) => browser.tabs.create({url}),
-    update: (url) => browser.tabs.update({url}),
+export const tabs = {
+  create:    (url) => browser.tabs.create({url}),
```

Comment:
nit but i'm not a big fan of manual alignment as it tends to get lost when using code format tools

Reply:
This is something  I do when I see a run of lines that are sufficiently similar. It helps me spot mistakes. I typically use rectangular edits to make subsequent changes, once the columns are aligned.

If it's a hard no, I'll untabulate it prior to the end.

Actually I was wondering if we still need the code in this file, as it seems like it's mostly just a wrapper around the browser extension APIs, and these seem to be somewhat moer stable now?

Reply:
Would definitely be nice to ditch the wrapper if it's no longer necessary. I think the only discrepancy I ever ran into was the `// Some weird bug in chrome...` comment (which is a terribly unhelpful comment 🤦 )